apps/gitness/src/components/Header.tsx

```diff
@@ -1,47 +1,193 @@
-import { useEffect, useState } from 'react'
-import { TopBarWidget, Project } from '@harnessio/playground'
-import { useNavigate } from 'react-router-dom'
+import { useEffect, useMemo, useState } from 'react'
+import { Avatar, AvatarFallback } from '@harnessio/canary'
+import {
+  TopBarWidget,
+  PipelineDropdownOption,
+  getInitials,
+  RepositoryDropdownOption,
+  ProjectDropdownOption
+} from '@harnessio/playground'
+import { useListExecutionsQuery, useListPipelinesQuery, useListReposQuery } from '@harnessio/code-service-client'
+import { useNavigate, useParams } from 'react-router-dom'
 import { useAppContext } from '../framework/context/AppContext'
 import { useGetSpaceURLParam } from '../framework/hooks/useGetSpaceParam'
+import { useGetRepoRef } from '../framework/hooks/useGetRepoPath'
+import { ExecutionDropdownOption } from '../../../../packages/playground/dist/components/common-breadcrumb/execution-dropdown'
+
+const CREATE_PROJECT = -2
```

Comment:
I'm curious about the - 2 set up here, may I know what is it for?

Reply:
obsolete code